--- src/build_xfs/xfs_mkfs.h	2006-08-14 18:14:50.000000000 +0200
+++ src/build_xfs/xfs_mkfs.h-new	2007-04-02 04:15:18.660018169 +0200
@@ -54,6 +54,8 @@
 						 * accept w/o warnings
 						 */
 
+#define	XFS_DFL_RTEXTSIZE	(4 * 1024)	/* 4KB */
+
 #define XFS_AG_BYTES(bblog)	((long long)BBSIZE << (bblog))
 #define	XFS_AG_MIN_BYTES	((XFS_AG_BYTES(15)))	/* 16 MB */
 #define XFS_AG_MIN_BLOCKS(blog)	((XFS_AG_BYTES(15)) >> (blog))
